gdk/static/cli_model.json

```diff
@@ -90,6 +90,20 @@
                     "--bucket"
                 ],
                 "help": "Name of the s3 bucket to use for uploading component artifacts."
+            },
+            "region": {
+                "name": [
+                    "-r",
+                    "--region"
+                ],
+                "help": "Name of the AWS region to use component creation."
+            },
+            "options": {
+                "name": [
+                    "-o",
+                    "--options"
+                ],
+                "help": "Extra configuration options used during component version creation."
```

Comment:
explain the format, ie. this needs to be a file path to a json file and then what is inside the json file?

Reply:
`gdk component publish -r us-west-2 -o "{\"file_upload_args\":{\"Metadata\":{\"key\":\"value\"}}}"`. It takes json string as the input. Will update the help text.